docs/phylum_auth_login.md

```diff
@@ -0,0 +1,10 @@
+---
+title: phylum auth login
+category: 6255e67693d5200013b1fa3e
+hidden: false
+---
+```sh
+phylum auth login
+```
+Login to an existing account.
```

Comment:
In some files, there is a brief summary of the command preceding its syntax, but in other files it follows the syntax.

Reply:
You can tell I was undecided. I have decided to put the summary line above and any additional context below. I believe this is all normalized now.